### PR TITLE
feat: add ignore url params option

### PR DESCRIPTION
**Pull request by carloschneider**
#78

### CODE REVIEW

**Review by nutboltu — CHANGES_REQUESTED** [review]

File: src/components/RequestItem/index.js [hunk 1]
```diff
@@ -111,7 +114,24 @@ export const RequestItem = ({
...
                     )}
                 </Field>
-                <Field />
+                <Field label="Ignore Params">
+                    <input
+                        type="radio"
+                        name="ignoreParams"
+                        value="true"
+                        onChange={onChangeHandler}
+                        checked={ignoreParams ? 'checked' : ''}
```
[the remark this answers is not in the document]
Comment (carloschneider) [reply src/components/RequestItem/index.js]: @nutboltu fixed.

**Review by nutboltu — APPROVED** [review]

**Comment by nutboltu** [comment]
@carloschneider there is some issue in the radio button group when I tried to click the radio for the first time. I am making a few changes to make it work.

**Comment by carloschneider** [comment]
@nutboltu Let me know if I can help with it

**Comment by nutboltu** [comment]
In the package, we are using `pathtoregexp` package to match the paths. I would recommend trying [this](https://github.com/pillarjs/path-to-regexp#zero-or-more) approach to resolve this issue.

As this is a special case, introducing a new field `Ignore params` will make confusion to others. Moreover, this is not a generalized API for this addon.

Considering the overall perception, I am going to revert the PR. Let's discuss further if the asterisk(*)  option would fit your requirement.

**Pull request merged**